kernel/mmanutils.c

```diff
@@ -693,6 +693,10 @@ int myst_release_process_mappings(pid_t pid)
                         ERAISE(-EINVAL);
 #endif
                     }
+                    else
+                    {
+                        myst_mman_pids_set(addr, len, 0);
```

Comment:
I'm thinking whether this can be done regardless myst_munmap succeeds or not (given that the fdmappings is freed already). It's unclear what's the expected behavior if myst_munmap fails here.